packages/superset-ui-connection/src/callApi/callApi.ts

```diff
@@ -26,6 +30,41 @@ export default function callApi({
     signal,
   };
 
+  if (method === 'GET' && cacheAvailable) {
+    return caches.open('superset').then(supersetCache =>
```

Comment:
the `superset` key should also be a constant in a separate file, and maybe more unique, matching the package name seems safer (even though it sounds like `Cache`s are scoped to by origin) e.g., `@SUPERSET-UI/CONNECTION`?

Reply:
Good point.